app/views/docs/oauth-providers/github.md

```diff
@@ -346,4 +346,7 @@ let account = Account(client)
 let session = try await account.updateSession(
     sessionId: "current"
 )
-```
\ No newline at end of file
+```
+## Accessing the Provider Access Token
```

Comment:
Can you include full code examples for each platform like other examples?

Reply:
I also included the code of how to use the access token in API request, let me know if its fine or if it's not needed